docs/presentation/content/CalculatedPropertiesSpecification.md

```diff
@@ -47,6 +48,22 @@ and [Ruleset Variables](../advanced/ECExpressions.md#ruleset-variables-user-sett
 
 ![Example of using "value" attribute](./media/calculatedpropertiesspecification-with-value-attribute.png)
 
+### Attribute: `type`
+
+Specifies return type of the calculated property. If a type that is not one of `"string" | "int" | "long" | "dateTime" | "boolean" | "bool" | "double"` is provided, calculated property won't appear in the property pane. If evaluated expression can't be converted to the specified type an error will be thrown in the frontend code.
```

Comment:
> If a type that is not one of `"string" | "int" | "long" | "dateTime" | "boolean" | "bool" | "double"` is provided, calculated property won't appear in the property pane.

I'd remove this, since it's a common behavior in presentation rules - when an invalid rule is specified, an error is logged and the rule is omitted.

> If evaluated expression can't be converted to the specified type an error will be thrown in the frontend code.

1. I'd mention that the library tries to map the type of ecexpression to the requested type.
2. I'd give an example of when it would fail.
3. We should avoid mentioning "frontend" here - the rules may be used on the backend as well. Instead, I think we should say that an error will be thrown when requesting content. Would be great to specify what the error will say.

Reply:
Updated comment

Reply:
> an error will be thrown when requesting content with the error message: `Encountered unexpected db result code: 1`

Not a great error message. Why doesn't it match what we set in our custom function implementation?

Reply:
Thes was due to our backend, it wasn't being returned. Added it and updated the comment.